src/sql/workbench/contrib/editData/browser/gridParentComponent.ts

```diff
@@ -72,31 +72,32 @@ export abstract class GridParentComponent {
 	protected _messageActive = true;
 	protected activeGrid = 0;
 
-	@ViewChildren('slickgrid') slickgrids: QueryList<SlickGrid>;
+
+	protected nativeElement: HTMLElement;
```

Comment:
please be consistent with variable prefixes within a particular class.  Below there are two protected variables, one with `_` prefix and one without.  They both should either have or not have the prefix in a particular file, and should be consistent with other variables of same visibility level with the file.  In this case, it looks like protected variables aren't using `_` so please remove from `_tables`.